win.Vagrantfile

```diff
@@ -0,0 +1,30 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.configure(2) do |config|
+  config.vm.box = "cliqz/cliqzfox-win10-builder_vmware"
+
+  config.vm.network "public_network"
+
+  config.vm.guest = :windows
+  config.vm.boot_timeout = 500
+  config.vm.communicator = "winrm"
+  config.winrm.username = "cliqzfoxer"
+  config.winrm.password = "cliqzfoxer-245"
+  config.winrm.timeout = 21600 # 6 hours
+
+  config.vm.provider "vmware_workstation" do |v|
+    v.gui = false
+    v.memory = 4048
```

Comment:
was planning parametrization for this one. 

my question is, should we handle release builds with this PR or create a separate PR just after this one is merged?